docs/api-and-integration/octopus.migrator.exe-command-line/partial-export.md

```diff
@@ -9,8 +9,56 @@ This command exports configuration data to a directory filtered by a single proj
 
 Usage:
 
-```bash
-Octopus.Migrator partial-export [<options>]
+```text
+Usage: octopus.migrator partial-export [<options>]
```

Comment:
These options look right to me :+1: But we've now got a repeated section beneath (under `Partial export options`).